src/GraphQl/Resolver/Factory/CollectionResolverFactory.php

```diff
@@ -49,8 +54,12 @@
     private $requestStack;
     private $paginationEnabled;
     private $resourceMetadataFactory;
+    /**
+     * @var EventDispatcherInterface
```

Comment:
I don't think the PHPDoc is needed.

Reply:
It’s not, indeed... It’s my PhpStorm’s default behavior, sorry for that. I’m fixing it.